### DISCUSSION TITLE
LEGO Mindstorms to be discontinued - What does this mean for pybricks?

### DISCUSSION THREAD

**Discussion by henriquedesousa** [pybricks/support#769] [General]
Hello, I'm a recent coming from the dark ages and since I'm a programmer, I'm looking for ways to activate my kids' mindset into learning a bit about logic using their beloved LEGO bricks.

However, time got in the way and I didn't got much time, until I read about the [discontinuing of Mindstorms](https://brickset.com/article/84219/lego-mindstorms-to-be-discontinued), which I never got not only because of time but because it's pretty expensive for what it offers (IMHO).

What do this community think about this? Will it hinder or help pybricks' development? And what about 2024 when the official app retires, how can we restore the original firmware?

Thanks in advance for your ideas.

**Comment by laurensvalk** [comment]
We will continue to support the MINDSTORMS hubs that we already support. We might even support older hubs, too.

Also note that LEGO Education SPIKE Prime uses the same hub (different color, same hub), so there should be at least one official LEGO app with official firmware updates and recovery after 2024.

But with Pybricks, even the less expensive hubs can do most of what you'd expect from a MINDSTORMS hub, so we're not worried about what you can build and program in the future :)
